src/main/java/pack/model/Company.java

```diff
@@ -0,0 +1,91 @@
+package pack.model;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.SequenceGenerator;
+
+import org.springframework.stereotype.Component;
+
+@Component
+@Entity
+public class Company {
+	@Id
+	@GeneratedValue(strategy=GenerationType.AUTO, generator="company_seq")
+	@SequenceGenerator(name="company_seq", sequenceName="company_seq",initialValue = 1, allocationSize = 100)
+	private int companyId;
+	@Column
+	private String name;
+	@Column
+	private String catchPhrase;
+	@Column
+	private String bs;
+	
+	
+	@OneToOne(cascade=CascadeType.ALL)
+	//@JoinColumn(name = "personId", nullable = false)
+	private Person personCompany;
```

Comment:
A company can have many persons. Please change this relation to use a one-to-many relationship.